<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSlot</class>
 <widget class="QWidget" name="AddSlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>СЛОТ</string>
  </property>
  <widget class="QLineEdit" name="IdSlot">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdStorage">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IdSlotLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Слота</string>
   </property>
  </widget>
  <widget class="QLabel" name="IdMaterialLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Матеріалу</string>
   </property>
  </widget>
  <widget class="QLabel" name="IdStorageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Складу</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MaterialCount">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddDeliveryButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Створити</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdMaterial">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MaterialCountLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Кількість матеріалу</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Створити новий слот</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Змінити  існуючий слот</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idSlotAdding">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="IdSlotLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Слота</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MaterialCountAdding">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="MaterialCountLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Зміна матеріалу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addMatherialsToSlot">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Додати</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
